modules/pom.xml

```diff
@@ -50,6 +50,12 @@
             <optional>true</optional>
             <exclusions>
                 <exclusion>
+                <!-- Infinispan might use a different version than Neo4j. It is 
+                    safe to exclude it as we are going to use a module already present on WildFly -->
```

Comment:
Are you sure you mean "already present on WildFly" ?
It is true that WildFly contains a copy of Infinispan (together with its dependencies) but we should be using the updated module from the `infinispan-wildfly-modules` artifacts. Let's make sure we're binding to the right version and clarify the comment.

Reply:
When we launch the tests we donwload and unzip the infinispan-wildfly-modules first so, technically, they are already in the WIldFly we are using :)

Reply:
I'm replacing it with:

                <!-- Infinispan might use a different version of Caffeine than Neo4j.
                    It is  safe to exclude it as we are going to use a module already
                    provided by the Infinispan modules for WildFly -->

ok?

Reply:
Sounds good.

For consistency, the same comment appear underneath this one in the class but instead of caffeine refers to lucene, could you update that as well?